src/assets/main.scss

```diff
@@ -16,6 +16,45 @@ body {
   text-rendering: optimizeLegibility;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  margin-bottom: 50px;
+}
+
+a,
+a:link,
+a:visited,
+a:hover,
+a:active {
+  text-decoration: none;
```

Comment:
In general it's a good design practice to keep some sort of non-color indication that element is a link at least on hover. It makes it easier for a user to spot them. And in those very specific cases like custom menu items you can override higher specificity styles

https://usability.yale.edu/web-accessibility/articles/links#non-color-indicators

Reply:
Good tip - thank you